client/src/Components/Feed/Feed.js

```diff
@@ -1,9 +1,12 @@
-import React from 'react';
+import * as React from 'react';
+import { useContext, useState, useEffect } from 'react';
```

Comment:
Not that it matters, but this can be consolidated into one line:
import React, { useContext, useState, useEffect} from 'react';
I'm not really sure what the * is for, do you know?